config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resource :users, only: [:create]
+
+  get "/session", to: "session#login"
+  get "/token_auth", to: "authentication#token_auth"
```

Comment:
We may want you to merge main into this branch first in order to resolve conflicts (?)